res/values/strings.xml

```diff
@@ -5,7 +5,9 @@
     <string name="short_description">Lightweight and privacy-conscious virtual keyboard for Android.</string>
     <string name="store_description">The main feature is that you can type more characters by swiping the keys towards the corners.\n\nThis application was originally designed for programmers using Termux.\nNow perfect for everyday use.\n\nThis application contains no ads, doesn\'t make any network requests and is Open Source.</string>
     <string name="pref_portrait">In portrait mode</string>
+    <string name="pref_portrait_unfolded">In portrait mode unfolded</string>
```

Comment:
Can you run `grade test` to update the other strings files ?

Reply:
Done